<launch>
    <!-- load configuration file -->
    <rosparam command="load" file="$(find visual_landing_provider)/config.yaml" />

    <rosparam param="base_path">file://$(find visual_landing_provider)</rosparam>

    <!-- Run node node -->
    <node pkg="visual_landing_provider" type="visual_infer.py" name="visual_infer_node" required="true" output="screen" />
</launch>
